app/Notifications/UserWelcome.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Notifications;
+
+use Carbon\Carbon;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Facades\Lang;
+
+/**
+ * This class provides the notification for newly created users with generated passwords.
+ *
+ * @package App\Notifications
+ */
+class UserWelcome extends Notification
+{
+    use Queueable;
+
+    /**
+     * The password reset token.
+     *
+     * @var string
+     */
+    private $token;
+
+    /**
+     * The date when the password will expire.
+     *
+     * @var Carbon
+     */
+    private $expireDate;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @param string $token
+     * @param Carbon $expireDate
+     */
+    public function __construct(string $token, Carbon $expireDate)
+    {
+        $this->token      = $token;
+        $this->expireDate = $expireDate;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed       $notifiable
+     * @return MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        $url = url('/reset_password?') . \Arr::query([
```

Comment:
Maybe add a url attribute so the view can show a new user specify text and not the regular password reset 
![image](https://user-images.githubusercontent.com/4281791/108712806-9c2fad80-7517-11eb-8831-eb1c3610dfdc.png)

Reply:
I solved this in 8432aad